platformio.ini

```diff
@@ -805,3 +806,21 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
+# Arduino(R) Nano ESP32
+# ------------------------------------------------------------------------------
+[env:arduino_nano_esp32]
+platform = espressif32@ ~6.4.0
```

Comment:
Sorry, overlooked one.
Please add this line, directly after the platform= line:

```ini
platform_packages =
```

Reason is that our [common] section defines platform packages for 8266. So as best practice, this variable should be cleared on esp32.

Reply:
Done.